frontend/src/metabase/dashboard/components/Dashboard/Dashboard.tsx

```diff
@@ -260,17 +275,19 @@ function DashboardInner(props: DashboardProps) {
 
   const canWrite = Boolean(dashboard?.can_write);
   const tabHasCards = currentTabDashcards.length > 0;
-  const dashboardHasCards = dashboard?.dashcards.length > 0;
+  const dashboardHasCards = dashboard && dashboard.dashcards.length > 0;
```

Comment:
How are these diff different - are we accounting for some edge cases here?

Reply:
Yep, slight difference. `dashboard?.dashcards.length`, when `dashboard = null`, is coerced to `null`, and typescript doesn't let us compare `null` and `0` when using `>`

Reply:
nice catch! that is indeed very subtle 🙏🏻